plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -25,7 +26,10 @@
 #include <boost/multi_index/member.hpp>
 #include <boost/multi_index/hashed_index.hpp>
 #include <boost/multi_index/ordered_index.hpp>
+#include <boost/multi_index/random_access_index.hpp>
 #include <boost/signals2/connection.hpp>
+#include <boost/range/adaptor/map.hpp>
+#include <boost/range/algorithm/copy.hpp>
```

Comment:
None of these new boost includes are needed.

Reply:
removed